### PR TITLE
new blog post productise_me

### PR DESCRIPTION
**Pull request by skinnybeans**

### CODE REVIEW

File: _posts/2017-09-26-productize_me.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - devops
+author: Henrik Axelsson
+excerpt:
+    Have you thought about how to make your work a product?
+---
+
+### Where we were
+A large client of ours has recently started their cloud migration journey. I was brought in as an iteration manager to help one of the recently renamed *Cloud Foundation* squads manage their work.
+
+The team had previously been known as *Technology Automation* team. Strangely enough their goal was trying to automate things for other teams as much as possible.
```
Comment (tpartington) [inline _posts/2017-09-26-productize_me.md]: should this be
'previously known as _the_ **Technology Automation** team'

File: _posts/2017-09-26-productise_me.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    - devops
+tags:
+    - devops
+author: Henrik Axelsson
+excerpt:
+    Have you thought about how to make your work a product?
+---
+
+### Where we were
+A large client of ours has recently started their cloud migration journey. I was brought in as an iteration manager to help one of the recently renamed *Cloud Foundation* squads manage their work.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: "had recently" -- make the tense the same as the rest of the document

File: _posts/2017-09-26-productise_me.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The team had previously been known as the *Technology Automation* team. Strangely enough their goal was trying to automate things for other teams as much as possible.
+
+### The challenge
+Unfortunately there were some problems with automation the team tried to provide in the past. Firstly, there was often changing priorities. Secondly, when things did get built, other teams didn't tend to use them.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: How about:

 Unfortunately there were some problems with the automation the team had tried to provide in the past: changing priorities often left work unfinished and then, when things did get built, other teams didn't tend to use them

File: _posts/2017-09-26-productise_me.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+In addition to this, it seemed like the team was doing a lot of manual work for other teams. Things like getting firewall changes done or setting up direct connect for AWS accounts. The team didn't actually do the task, but knew who could so often took it upon themselves to help coordinate these activities.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: Direct Connect is a proper noun and should be capitalised.

"The team didn't actually do the task, but knew who could, so often took..." (oxford comma FTW)

File: _posts/2017-09-26-productise_me.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+#### What is a service?
+Here I am using the definition of a service in the traditional sense. For example: *helping get a firewall change done*. Not in the *Software as a Service (SaaS)* sense.
+
+
+#### What is a product?
+A techy thing that has to be built, that will be used by a customer. This could be SaaS or some other kind of consumable digital product.
+
+### Focus
+While doing both building products and providing services could possibly be done by the team, I felt that better outcomes would be achieved by focusing on one or the other.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: The sentence bravely stumbled on, waving its arms for balance, until it tripped over the lone comma and fell face-forwards, its momentum dragging it forwards, leaving a bloody smear until it arrived at the full stop.

How about:
"While the team could both build products and deliver services, I felt that they would be able to deliver better outcomes by focusing on one task exclusively" (or something)

File: _posts/2017-09-26-productise_me.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+The view from management was the team should be building things that other teams can use. Having this direction was very helpful, as it meant the service aspect could be reduced to supporting the products that we would build.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: was _that_ the team ...

File: _posts/2017-09-26-productise_me.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+For the cloud migration initiative, the customers were teams across the organisation that were:
+* Moving applications in the cloud.
+* Building new applications in the cloud.
+* Storing data in the cloud.
+
+We needed to make their life easier. We needed to build products that they wanted to use.
+
+### Putting the customer first
+So far the team has built three products since Cevo engagement started.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: s/has/had/

This is the first time you mention the Cevo name; the rest has been first-person. Perhaps either use the Cevo name further up, or make this be first-person as well.

File: _posts/2017-09-26-productise_me.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+* Is it easy to use?
+* Is it easy to maintain?
+
+### Woodstock
+
+I'll use one of the products built as an example. We called it Woodstock and it's used aggregate CloudTrail logs.
+
+Why Woodstock? Well that's were all the logs are kept! The accounts that send logs to Woodstock are called Lumberjacks. While the names may sound silly, it's one more thing that helps give life to what we built. Our customers always remember it too!
+
+At our client, any team running a workload in AWS must send their CloudTrail logs back on premise so they can be analysed for suspicious activity. Prior to Woodstock, each workload owner had to spend several weeks liaising with security to get Splunk integration sorted out.
```
Comment (nonspecialist) [inline _posts/2017-09-26-productise_me.md]: "on premises" actually. "On premise" would be "based on the assumption"

File: _posts/2017-09-26-productise_me.md (hunk 5 above)
Comment (skinnybeans) [reply _posts/2017-09-26-productise_me.md]: Best review comment ever!

Comment (nonspecialist) [reply _posts/2017-09-26-productise_me.md]: gah! now I re-read it, I used `forwards` twice. Ick. I'd replace the second one to read "its momentum dragging it on"

Comment (nonspecialist) [reply _posts/2017-09-26-productise_me.md]: no wait, I used `on` already ! I'm a failure of a grammar schoolmarm.

"its momentum dragging it in a bloody smear only punctuated by the full stop."

Ok, I'm done.

File: _posts/2017-09-26-productise_me.md (hunk 8 above)
Comment (skinnybeans) [reply _posts/2017-09-26-productise_me.md]: Damn the english language..

File: _posts/2017-09-26-productise_me.md (hunk 7 above)
Comment (skinnybeans) [reply _posts/2017-09-26-productise_me.md]: I'll mention Tom and Cevo in the into paragraph too.

File: _posts/2017-09-26-productise_me.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+We have focused carefully on the user experience, in particular:
+* Is it easy to install?
+* Is it easy to use?
+* Is it easy to maintain?
+
+### Woodstock
+
+I'll use one of the products built as an example. We called it Woodstock and it's used aggregate CloudTrail logs.
+
+Why Woodstock? Well that's were all the logs are kept! The accounts that send logs to Woodstock are called Lumberjacks. While the names may sound silly, it's one more thing that helps give life to what we built. Our customers always remember it too!
```
Comment (stevemac007) [inline _posts/2017-09-26-productise_me.md]: Were --> where

File: _posts/2017-09-26-productise_me.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+### Focus
+While the team could both build products and deliver services, I felt that they would be able to deliver better outcomes by focusing on one task exclusively.
+
+The view from management was that the team should be building things that other teams can use. Having this direction was very helpful, as it meant the service aspect could be reduced to supporting the products that we would build.
+
+As the team had tried to build things for other teams in the past and not had the success they were hoping for, how could we increase our chances of hitting the mark this time?
+
+### Product centric view
+I decided that software/infrastructure/SaaS/scripts/automation that the team would provide should be treated as any other product:
```
Comment (stevemac007) [inline _posts/2017-09-26-productise_me.md]: That ...... That??

I decided the .... That - I think that's what u meant

**Pull request merged**